examples/node/scripts/createDAPostViaTypedData.ts

```diff
@@ -0,0 +1,69 @@
+import { isCreateDataAvailabilityPublicationResult } from "@lens-protocol/client";
+import { getAuthenticatedClient } from "./shared/getAuthenticatedClient";
+import { setupWallet } from "./shared/setupWallet";
+import { uploadWithBundlr } from "./shared/uploadWithBundlr";
+import { getActiveProfile } from "./shared/getActiveProfile";
+import { buildPublicationMetadata } from "./shared/buildPublicationMetadata";
+
+async function main() {
+  const wallet = setupWallet();
+  const address = await wallet.getAddress();
+  const lensClient = await getAuthenticatedClient(wallet);
+  const profile = await getActiveProfile(lensClient, address);
+  const profileId = profile.id;
+
+  // prepare metadata
+  const metadata = buildPublicationMetadata({
+    content: "Data Availability Post created with LensClient SDK",
+    name: "Data Availability Post created with LensClient SDK",
+  });
+
+  // validate metadata
+  const validateResult = await lensClient.publication.validateMetadata(metadata);
+
+  if (!validateResult.valid) {
+    throw new Error(`Metadata is not valid.`);
+  }
+
+  // upload metadata to
+  const contentURI = await uploadWithBundlr(metadata);
+
+  console.log(`Post metadata was uploaded to ${contentURI}`);
+
+  // fetch a create DA post typed data
+  const createPostTypedDataResult =
+    await lensClient.publication.createDataAvailabilityPostTypedData({
+      from: profileId,
+      contentURI,
+    });
+
+  const createPostTypedDataValue = createPostTypedDataResult.unwrap();
+
+  console.log(`createDataAvailabilityPostTypedData result: `, createPostTypedDataValue);
+
+  // sign with the wallet
+  const signedTypedData = await wallet._signTypedData(
+    createPostTypedDataValue.typedData.domain,
+    createPostTypedDataValue.typedData.types,
+    createPostTypedDataValue.typedData.value
+  );
+
+  console.log(`Broadcasting signed createDataAvailabilityPostTypedData...`);
+
+  const broadcastResult = await lensClient.transaction.broadcastDataAvailability({
+    id: createPostTypedDataValue.id,
```

Comment:
I think we can destructure all variables 🙇🏼

Something like 

```
const { id, typedData } = createPostTypedDataResult.unwrap();
```

Reply:
this is an example script for integrators to understand how to use the SDK in different scenarios, I assumed the more meaningful names of consts/vars, the more self-explanatory the example is. I don't think restructuring makes any difference here, for sure it doesn't improve the clarity of the example. In the commented line of code, it is clear where the `id` is supposed to come from.